internal/engine/selection.go

```diff
@@ -27,19 +27,19 @@ func (e Engine) evaluateSelection(ctx ExecutionContext, sel command.Select) (tab
 	case command.EqualityExpr, command.GreaterThanExpr, command.GreaterThanOrEqualToExpr, command.LessThanExpr, command.LessThanOrEqualToExpr:
 	}
 
-	return table.NewFilteredRow(origin, func(r table.RowWithColInfo) bool {
+	return table.NewFilteredRow(origin, func(r table.RowWithColInfo) (bool, error) {
 		defer e.profiler.Enter("selection (lazy)").Exit()
```

Comment:
Magic strings? (`selection (lazy)`)

Reply:
ok for profiling I think
this is just for developing, and won't appear anywhere else, or for a user